README.md

```diff
@@ -178,8 +178,8 @@ First, clone the repo and then startup our local dev environment:
 
 ```sh
 $ git clone git@github.com:segmentio/analytics-next.git
-$ cd analytics-next
-$ make dev
+$ cd analytics-next && yarn
+$ yarn dev
```

Comment:
I think the unit testing section needs to be updated to `yarn test` from `make test-unit` as well.

Reply:
Good fallout. README and  other build behavior should actually have tests, I can write some